assembly/gradle-plugin/src/site/markdown/index.md.vm

```diff
@@ -17,6 +17,14 @@ buildscript{
         classpath '${antennaMavenGroupId}:${antennaGradlePluginName}:VERSION'
     }
 }
+plugins {
+    id 'java'
+}
+
+tasks.withType(Jar) {
+    destinationDir = file("$rootDir/build")
+}
+version '1.0-SNAPSHOT'
```

Comment:
Why is the `group 'org.eclipse.sw360.antenna'` not necessary, which was added in `example-projects/example-project/build.gradle`?

It would also be good if the order of the blocks in this file and the examples would match to simplify it for users.

Reply:
Is not necessary in general, I will remove it.